pkg/plugin/plugin.go

```diff
@@ -21,6 +21,8 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
+	//nolint
```

Comment:
why is this needed, do you have the golangci-lint error handy? I don't see the lint error https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_aws-encryption-provider/89/pull-aws-encryption-provider-verify/1598078085045948416

Reply:
> why is this needed, do you have the golangci-lint error handy? I don't see the lint error https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_aws-encryption-provider/89/pull-aws-encryption-provider-verify/1598078085045948416

The `golangci-lint` is complaining about `awserr` not used, which is actually used, there's similar thread about wrong judgement from `golangci-lint`, https://github.com/golangci/golangci-lint/discussions/2214, someone claimed to fix some failure with golang version bump from 1.15 to 1.16, however, `aws-encryption-provider` has a hard limit on golang 1.15 as the highest version due to protobuf.